buildscripts/templates/registry/incubating_java/weaver.yaml

```diff
@@ -0,0 +1,60 @@
+comment_formats:
+  javadoc:
+    format: html
+    header: "/**"
+    prefix: " * "
+    footer: " */"
+    old_style_paragraph: true
+    omit_closing_li: true
+    inline_code_snippet: "{@code {{code}}}"
+    block_code_snippet: "<pre>{@code {{code}}}</pre>"
+    trim: true
+    remove_trailing_dots: false
+default_comment_format: javadoc
+templates:
+  - pattern: IncubatingSemanticAttributes.java.j2
+    filter: > 
+      semconv_grouped_attributes({
+        "exclude_root_namespace": ["ios", "aspnetcore", "signalr"], 
+        "exclude_stability": ["deprecated"],
+        "exclude_deprecated": true
```

Comment:
@jack-berg - This is why we don't see deprecated *and* experimental things.  Setting to false, they'll all come back.

Reply:
These should be back now.